cmd/controller/main.go

```diff
@@ -7,6 +7,7 @@ package main
 
 import (
 	"flag"
+	"go.uber.org/zap/zapcore"
```

Comment:
this should go into below section with other non-std dependencies.

Reply:
Done with our last change.